go.mod

```diff
@@ -1,10 +1,19 @@
 module github.com/maxmind/mmdbwriter
 
-go 1.14
+go 1.18
```

Comment:
I think we will want 1.17 here as it we want to support it.

Reply:
@oschwald 
Also fixed